node/how_to_with_npm/apollo.md

```diff
@@ -0,0 +1,190 @@
+# How to use Apollo with Deno
+
+[Apollo Server](https://www.apollographql.com/) is a GraphQL server that you can
+set up in minutes and use with your existing data source (or REST API). You can
+then connect any GraphQL client to it to receive the data and take advantage of
+GraphQL benefits, such as type-checking and efficient fetching.
+
+We’re going to get a simple Apollo server up and running that will allow us to
```

Comment:
```suggestion
We're going to get a simple Apollo server up and running that will allow us to
```

We should maybe do a find and replace for all `’` to `'`

Reply:
good idea!